### ISSUE TITLE
dnn_dropout in DNNLinearCombinedClassifier doesn't  work

### ISSUE THREAD

**Issue by 333caowei** [tensorflow/tensorflow#21357]
I use DNNLinearCombinedClassifier to train a wide & deep model with dnn_dropout=0.5。
```
m = tf.estimator.DNNLinearCombinedClassifier(
    model_dir=model_dir,
    linear_feature_columns=crossed_columns,
    dnn_feature_columns=deep_columns,
    dnn_dropout=0.5,
    dnn_hidden_units=[512, 256, 256, 128])

m.train(
    input_fn=input_fn(train_file_name, num_epochs=None, shuffle=True, 
    is_eval=False,steps=train_steps, 
    hooks=[logger_hook])
```

 Then I evaluate model as follow: 
```
m = tf.estimator.DNNLinearCombinedClassifier(
    model_dir=model_dir,
    linear_feature_columns=crossed_columns,
    dnn_feature_columns=deep_columns,
    dnn_dropout=1.0,
    dnn_hidden_units=[512, 256, 256, 128])

results = m.evaluate(
    input_fn=input_fn(test_file_name, num_epochs=1, shuffle=False, is_eval=True),
    steps=None)
```
I find no matter I set dnn_dropout=0.1 or dnn_dropout=1.0, the result are same. It seems that this parameter doesn't work.
My version is 1.4

**Comment by jony0917** [comment]
TF will raise an ValueError exception if dnn_dropout=1.0, did you forget to call train at second time ? Anyway you can check it by using two different model_dir.

**Comment by 333caowei** [comment]
@jony0917 Trying dnn_dropout=0.1 and 0.8, whatever any kind of droupout ratio can get the same result.

**Comment by 333caowei** [comment]
Can anyone help？

**Comment by tensorflowbutler** [comment]
Nagging Assignee @shivaniag: It has been 29 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @shivaniag: It has been 44 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @shivaniag: It has been 59 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @shivaniag: It has been 89 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by Git-David** [comment]
I'm also waiting for the answer.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @shivaniag: It has been 14 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Assignee @shivaniag: It has been 29 days with no activity and this issue has an assignee. Please update the label and/or status accordingly.

**Comment by tilakrayal** [comment]
@333caowei ,
We see that you are using tf version 1.4.0, 1.x is not actively supported, please update to 2.x and let us know if you are facing same issue.Thanks!
